kubernetes/samples/scripts/domain-lifecycle/stopServer.sh

```diff
@@ -153,6 +154,16 @@ else
   fi
 fi
 
+if [[ -n "${clusterName}" && "${keepReplicaConstant}" == 'false' ]]; then
+  # check if replica count can decrease below current value
+  isReplicaCountEqualToMinReplicas "${domainJson}" "${clusterName}" replicasEqualsMinReplicas
+  if [ "${replicasEqualsMinReplicas}" == 'true' ]; then
+    printInfo "Current replica count value is same as minimum number of replica count. \
```

Comment:
Maybe provide a little more detail:

Not decreasing replica count: it is at its minimum. (See 'domain.spec.allowReplicasBelowMinDynClusterSize' and 'domain.status.clusters[].minimumReplicas' for details.)

Reply:
Changed the message in 6dae3f1. Please let me know if more details are needed.